core/domain/subscription_services.py

```diff
@@ -57,14 +57,14 @@ def subscribe_to_activity(user_id, activity_id):
     if not subscriptions_model:
         subscriptions_model = user_models.UserSubscriptionsModel(id=user_id)
 
-    if (activity_id not in
-            subscriptions_model.activity_ids):
+    if activity_id not in subscriptions_model.activity_ids:
         subscriptions_model.activity_ids.append(activity_id)
         subscriptions_model.put()
 
 
 def get_activity_ids_subscribed_to(user_id):
-    """Returns a list with all subscriptions' ids of given user.
+    """Returns a list with ids of all activities (i.e., explorations and
```

Comment:
Yes, the comment is wrong unless we want to update subscription_services to actually store activities, rather than explorations and collections individually. I think for the sake of time, we might as well just store them separately. In the future, we might want to consider unifying them and performing a migration. Also, fixed the comment. I updated all references to 'activity' to instead refer to 'exploration' in the cases where it did not refer to both a collection and exploration.